packages/dd-trace/src/plugins/util/ip_blocklist.js

```diff
@@ -21,16 +21,23 @@ if (semver.satisfies(process.version, '>=14.18.0')) {
 
     check (address, type) {
       try {
-        const ip = ipaddr.process(address)
+        let ip = ipaddr.parse(address)
 
-        if (type === 'ipv4' || ip.isIPv4MappedAddress()) {
-          for (const range of this.v4Ranges) {
+        type = ip.kind()

```

Comment:
`type` parameter is not used anymore, can we remove the param from the line 22?

Reply:
well, I left it because it's still sent by the caller, since it's a polyfill, it's supposed to have the same API, but internally I don't use it. I just left it for clarification

Reply:
wdyt ?

Reply:
like I still call it like this: `check(ip, ipType)` so it's weird to remove the argument from the function declaration imo

Reply:
Ok, if you use `ipType` the linter will be tell you that you have an unused var. Ok for me to maintain it as you have in the PR